linkerd/app/outbound/src/http/logical/policy/route.rs

```diff
@@ -176,6 +169,16 @@ impl<T> filters::Apply for Http<T> {
     }
 }
 
+impl<B, T> svc::ExtractParam<metrics::labels::Route, http::Request<B>> for Http<T> {
+    fn extract_param(&self, req: &http::Request<B>) -> metrics::labels::Route {
+        metrics::labels::Route::new(
+            self.params.parent_ref.clone(),
+            self.params.route_ref.clone(),
+            req.uri(),
+        )
+    }
+}
```

Comment:
i'm happy to see this. honestly, this was how i intuitively expected things to work when i first saw how we route requests.

Reply:
https://github.com/linkerd/linkerd2-proxy/pull/3604#discussion_r1949951889 and lends us this pleasant change!